Gemfile

```diff
@@ -19,6 +19,7 @@ group(:test) do
   gem('rubocop-performance', require: false)
   gem('rubocop-sorbet', require: false)
   gem('webmock')
+  gem('simplecov', require: false)
```

Comment:
Could this be added to a separate PR, please?

Reply:
Sure thing